### PR TITLE
Add script to query CISA Exploited Vuln DB and add to GSD DB

### PR DESCRIPTION
**Pull request by oswalpalash**
Addresses #15 
Update to GSD Database : https://github.com/cloudsecurityalliance/gsd-database/pull/2250/files

### CODE REVIEW

**Review by raphaelahrens — COMMENTED** [review]
A good commit.
As you mentioned in your TODO there is some room for improvement by extracting common  functionality.
But this will require some reconstruction, which makes more sense at a later point.

File: securitylist/src/securitylist/CISA.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    json_data = json.loads(original_data)
+    namespace = 'cisa.gov'
+    try:
+      if (json_data['namespaces'][namespace]) != json_blob:
+         json_data['namespaces'][namespace] = json_blob
+         print(filename + " updated")
+    except KeyError:
+      json_data['namespaces'][namespace] = json_blob
+    with open(filename, 'w') as fh:
+      fh.write(json.dumps(json_data,sort_keys=True, indent=4))
```
[the remark this answers is not in the document]
Comment (oswalpalash) [reply securitylist/src/securitylist/CISA.py:48]: thanks for this! I changed it slightly `json.dump(json_data, fh, sort_keys=True, indent=4)`

**Comment by oswalpalash** [comment]
Thanks for  the review @raphaelahrens ! I have made the changes you recommended! Gave it a few test runs on the GSD database and it works correctly.

**Review by joshbressers — APPROVED** [review]

**Comment by joshbressers** [comment]
This is great, thanks!

**Pull request merged**